### DISCUSSION TITLE
Capitalize claim

### DISCUSSION THREAD

**Discussion by ripleymj** [OpenIDC/mod_auth_openidc#1236] [Q&A] [answered]
Hello, I'm very new to this, so please be gentle with what's probably a stupid question. I'm trying to transform a claim from username@domain.com to just USERNAME for a proxied application. It seems like this should work:

OIDCRemoteUserClaim preferred_username ^(.*)@.*$ \U$1

but that gives me all sorts of errors and looks like it ends in a segfault. I can get the sample config of "^(.*)@(.*) $2\\$1" to work, so I'm feel like I'm near the right place but clearly missing something. Is it even possible to use the substitution string to capitalize? This is with the RedHat 8 packaged version 2.4.9.4.

Thanks for any advice you can offer!

**Comment by zandbelt** [comment, marked as answer]
you'll need to use a newer version, it can be downloaded from the Assets section of the Releases page https://github.com/OpenIDC/mod_auth_openidc/releases/

**Reply by ripleymj** [reply]
Thanks for the quick reply!
